cmake/projects/drm/hunter.cmake

```diff
@@ -22,7 +23,17 @@ hunter_add_version(
 )
 
 hunter_configuration_types(drm CONFIGURATION_TYPES Release)
-hunter_pick_scheme(DEFAULT drm)
+hunter_pick_scheme(DEFAULT url_sha1_autotools)
+list(APPEND drm_dependencies
```

Comment:
Actually it's vice versa: I did remove guards because it can be processed several times. It's a part of so-called "CACHE RUN" when we don't download package but looking up for cached entry. We have to calculate version and arguments before.